lib/_http_server.js

```diff
@@ -348,6 +349,28 @@ Server.prototype.setTimeout = function setTimeout(msecs, callback) {
   return this;
 };
 
+Server.prototype[EE.captureRejectionSymbol] = function(
+  err, event, ...args) {
+
+  switch (event) {
+    case 'request':
+      const [ , res] = args;
+      if (!res.headersSent && !res.finished) {
```

Comment:
I would probably prefer `writableFinished` over `finished` here since `finished` in http2 compat means something else.